prototype2/adc_readout/AdcParse.cpp

```diff
@@ -79,8 +89,6 @@ HeaderInfo parseHeader(const InData &Packet) {
     ReturnInfo.Type = PacketType::Data;
   } else if (0x2222 == Header.PacketType) {
```

Comment:
Hardcoded protocol values?

Reply:
I removed the magic values. They are now defined at the top of the implementation-file.